### PR TITLE
profile_tasks README.md

### PR DESCRIPTION
**Pull request by aioue**
As agreed with Brian Coca in Ansible Project group:
https://groups.google.com/forum/#!searchin/ansible-project/tom$20paine/ansible-project/__nv6BZs2yU/AkYQ0HU-BQAJ

### CODE REVIEW

**Comment by bcoca** [comment]
I thought we were making them .rst

**Comment by bcoca** [comment]
cc @docschick this is one way we are thinking of documenting the 'non module' plugins, the other thought was to embed this in the plugins themselves.

also core decided to standarize on RST so this should be changed from markdown.

**Comment by aioue** [comment]
Will convert to RST asap.

**Comment by docschick** [comment]
@bcoca I think this way of documenting the non-module plugins will work, but just to make sure I understand the way these will be pulled in and displayed, will these end up part of the overall core docs that we build or just as a README for the users to go and find?

**Comment by bcoca** [comment]
I hope that, in the end, both and also accessible through ansible-doc

**Comment by aioue** [comment]
@bcoca updated.

**Comment by docschick** [comment]
@bcoca nice! sounds like a good plan to me! +1

**Comment by docschick** [comment]
We might want to change the file extension from .md to .rst as well. :-)

**Comment by aioue** [comment]
@docschick done!

File: lib/ansible/plugins/callback/profile_tasks.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Mashup of 2 excellent original works:
+
+-  (https://github.com/jlafon/ansible-profile)
+-  (https://github.com/junaid18183/ansible_home/blob/master/ansible_plugins/callback_plugins/timestamp.py.old)
+
+Usage
+-----
+
+Add ``profile_taks`` to the ``callback_whitelist`` in ``ansible.cfg``.
```
Comment (jhawkesworth) [inline lib/ansible/plugins/callback/profile_tasks.rst]: typo? 'profile_taks' -> 'profile_tasks'?

**Comment by aioue** [comment]
@jhawkesworth fixed.

**Comment by aioue** [comment]
@docschick ready for review

**Comment by docschick** [comment]
+Mashup of 2 excellent original works:
+
+-  (https://github.com/jlafon/ansible-profile)
+-  (https://github.com/junaid18183/ansible_home/blob/master/ansible_plugins/callback_plugins/timestamp.py.old)

Question about the above--why are there " ( ) " around the URLs?
Other than that, looks fine with me. No glaring errors or typos. Clear and concise.  I could be nitpicky, but it isn't necessary. :-)   :+1:

**Comment by aioue** [comment]
@docschick legacy from the original markdown. Fixed and ready for review again.

**Comment by docschick** [comment]
:+1:

**Pull request merged**